apps/bblcricket/bbl_cricket.star

```diff
@@ -0,0 +1,771 @@
+                        BatsmanScore(Batsman1, Batsman1_Runs_Str, BatsmanColor),
+                        BatsmanScore(Batsman2, Batsman2_Runs_Str, Batsman2Color),
+                        StatusRow(T20_Status1, T20_Status1Color),
+                    ],
+                ),
+                render.Column(
+                    children = [
+                        TeamScore(BattingTeam, BattingTeamColor, Wickets, Runs),
+                        BatsmanScore(Batsman1, Batsman1_Runs_Str, BatsmanColor),
+                        BatsmanScore(Batsman2, Batsman2_Runs_Str, Batsman2Color),
+                        StatusRow(T20_Status2, T20_Status2Color),
+                    ],
+                ),
+                render.Column(
+                    children = [
+                        TeamScore(BattingTeam, BattingTeamColor, Wickets, Runs),
+                        BatsmanScore(Batsman1, Batsman1_Runs_Str, BatsmanColor),
+                        BatsmanScore(Batsman2, Batsman2_Runs_Str, Batsman2Color),
+                        StatusRow(T20_Status3, T20_Status3Color),
+                    ],
+                ),
+                render.Column(
+                    children = [
+                        TeamScore(BattingTeam, BattingTeamColor, Wickets, Runs),
+                        BatsmanScore(Batsman1, Batsman1_Runs_Str, BatsmanColor),
+                        BatsmanScore(Batsman2, Batsman2_Runs_Str, Batsman2Color),
+                        StatusRow(T20_Status4, T20_Status4Color),
+                    ],
+                ),
+            ]
+
+            return render.Root(
+                delay = int(4000),
+                child = render.Animation(children = renderScreens),
+            )
+
+            # Game is coming up
+        elif Match_JSON["match"]["stage"] == "SCHEDULED" or "PRE":
+            # Who is playing who
+            Team1_Name = Match_JSON["match"]["teams"][0]["team"]["name"]
+            Team2_Name = Match_JSON["match"]["teams"][1]["team"]["name"]
+            Team1_ID = Match_JSON["match"]["teams"][0]["team"]["id"]
+            Team2_ID = Match_JSON["match"]["teams"][1]["team"]["id"]
+
+            Team1_Color = getTeamFontColor(Team1_ID)
+            Team2_Color = getTeamFontColor(Team2_ID)
+
+            # Get the time of the game in the user's timezone
+            StartTime = Match_JSON["match"]["startTime"]
+            ParseTime = time.parse_time(StartTime, format = "2006-01-02T15:04:00.000Z")
+            MatchTimezone = Match_JSON["match"]["ground"]["town"]["timezone"]
+
+            location = DEFAULT_LOCATION
+            loc = json.decode(location)
+            timezone = loc["timezone"]
+
+            MyTime = time.parse_time(StartTime, format = "2006-01-02T15:04:00.000Z").in_location(timezone)
+            Time = MyTime.format("15:04")
+            Date = MyTime.format("Jan 2")
+
+            Standings_JSON = http.get(Standings_URL).json()
```

Comment:
Caching here too.

Reply:
This one should be a `get_cachable_data()`, right?

Reply:
Or just removed entirely since you're geting `Standings_JSON` above on L536?

Reply:
Yes, we can remove it as its already defined earlier on

Reply:
I've commented it out now